isb_lib/sitemaps/thing_sitemap.py

```diff
@@ -31,15 +33,13 @@ def __init__(
         max_length: int,
         things: typing.List[typing.Dict[typing.AnyStr, typing.AnyStr]],
     ):
-        self._things = things
+        self._things: list = things
```

Comment:
I didn't know you could do this with iVars, but this is the way to do it according to the mypy reference.